src/components/Person/Person.jsx

```diff
@@ -1 +1,35 @@
-// Write code here
+import React from 'react';
+import './Person.scss';
+
+export const Person = ({
+  name,
+  age,
+  sex,
+  isMarried,
+  partnerName,
+}) => {
+  const partner = (sex === 'm')
```

Comment:
redundant parentheses
```suggestion
  const partner = sex === 'm'
```